### ISSUE TITLE
Support recipes with conflicting inputs

### ISSUE THREAD

**Issue by Skreperiukas** [CyclopsMC/IntegratedCrafting#100]
#### Issue type:

- :heavy_plus_sign: Feature request 

____

#### Short description:
Items with same recipe can't be autocrafted.

#### Steps to reproduce the problem:

1. Find items that share the same recipe (In my case it's @ValhelsiaFurniture Table and Chair && @Minecraft Wooden Slab).
2. Setup variable card with Wooden Slab recipe.
3. Insert variable card into Crafting Interface.
4. Crafting interface shows X mark and terminal does not have Wooden Slab autocrafting as an option.

#### Expected behaviour:
Network let's me craft the desired item setup according to Variable Card.

____

#### Versions:

- This mod: 1.1.6
- Minecraft: 1.20.1
- Forge: 47.2.1

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
If you have equal inputs for the same recipe, then something must be wrong in your pack.
AFAIK, vanilla doesn't allow multiple equal-input recipes, and will just pick the first matching one when used.
Manually using the crafting table should produce the same issue.

Feel free to let me know if I misunderstood somehow.

**Comment by Skreperiukas** [comment]
In vanilla yes. Modded sometimes has same recipes for multiple items and its usually fixed using @Polymorph mod which adds a button in crafting table that lets you choose which outcome you want. I reported this because in larger modpacks this is inevitable and autocrafting for some items becomes impossible. I don't know how preciselly it should be fixed, but I think it's possible? @AppliedEnergystics2 has that figured out.

**Comment by rubensworks** [comment]
I see, thanks for the clarification @Skreperiukas.

I'll convert this issue to a feature request.

**Comment by nickbanana** [comment]
sorry but any progress on this?

**Comment by rubensworks** [comment]
Nope, no one is working on this AFAIK, but feel free to submit a PR.
